src/main/java/org/bricolages/streaming/filter/TextOp.java

```diff
@@ -65,4 +65,15 @@ String castStringForce(Object value) {
         if (!(value instanceof String)) return null;
         return (String)value;
     }
+
+    final Pattern nullCharsPattern = Pattern.compile("^\\x00+$");
+    String removeHeadNullChars(String str) {
```

Comment:
I thought `\0hoge` should be `hoge`. Because I thought it's better to preserve as much data as possible in this phase and allow users to choose whether ignore this data or not.

Why whole text should be dropped? Is that because `\0hoge` and `hoge` are not same? How about change to `\\0hoge`?